apps/docs/src/guide/predicates/methods.md

```diff
@@ -0,0 +1,52 @@
+# Interacting With Predicates
+
+The `Predicate` class extends the [`Account`](https://docs.fuel.network/docs/fuels-ts/account/) class, and therefore it inherits all of it's methods.  Therefore, there are multiple ways to interact with predicates, but broadly speaking there are 3 general use cases that would be of interest to a user. We can think of them in terms of `Checking Balances`, `Transactions` and `Transfers`
+
+
+## Checking Balances
+
+### `getBalances`
+
+This will return the balances of the wallet that is being used to interact with the predicate.
+
+<<< @/../../docs-snippets/src/guide/wallets/checking-balances.test.ts#checking-balances-2{ts:line-numbers}
+
+### `getResourcesToSpend`
+
+This will return the resources that will be spent when calling the predicate. This is useful when creating the predicate transaction so that you can ensure that the wallet has sufficient funds to cover the transaction fees.
```

Comment:
```suggestion
This will return the resources that are spent when calling the predicate. This is useful when creating the predicate transaction to ensure that the wallet has sufficient funds to cover the transaction fees.
```